src/components/RouterBreadcrumbs.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import Breadcrumbs from '@material-ui/lab/Breadcrumbs';
+import Link from '@material-ui/core/Link';
+import Typography from '@material-ui/core/Typography';
+import { Link as RouterLink } from 'react-router-dom';
+import { Paper, withStyles } from '@material-ui/core';
+
+const styles = (theme) => ({
+  paper: {
+    padding: `${theme.spacing.unit}px ${theme.spacing.unit * 2}px`,
```

Comment:
Please for comment why we do `* 2` here.

Reply:
Removing this actually. Took it from the docs on `Breadcrumbs`, but isn't really necessary.